common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +97,876 @@ innovation_taming_the_skies = {
 
 	potential = {
 		is_giant_culture_trigger = no
+		NOT = {
+			has_aerial_maa_trigger = yes
+		}
 	}
 
 	unlock_maa = aerial_unit
 
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @archmage
+
+	potential = {
+		AND = {
+			OR = {
+				this.culture_group = culture_group:human_group
+				this = culture:blood_elf
+				this = culture:high_elf
+			}
+			NOT = {
+				this = culture:pirate_culture
+				this = culture:wastewander	
+			}
+		}
+	}
+
+	unlock_maa = archmage
+
+	flag = global_maa
+}
+
+
+innovation_blademaster = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @horde_special
+
+	potential = {
+		this = culture:burning_blade
+	}
+
+	unlock_maa = blademaster
+
+	flag = global_maa
+}
+
+innovation_buccaneer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @buccaneer
+
+	potential = {
+		OR = {
+			this = culture:pirate_culture
+			this = culture:blackwater
+		}
+	}
+
+	unlock_maa = buccaneer
+
+	flag = global_maa
+}
+
+innovation_brute = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @brute
+
+	potential = {
+		this.culture_group = culture_group:gnoll_group
+	}
+
+	unlock_maa = brute
+
+	flag = global_maa
+}
+
+innovation_candleholder = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @candleholder
+
+	potential = {
+		this.culture_group = culture_group:kobold_group
+	}
+
+	unlock_maa = candleholder
+
+	flag = global_maa
+}
+
+innovation_dervishes = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dervishes
+
+	potential = {
+		this = culture:wastewander
+	}
+
+	unlock_maa = dervishes
+
+	flag = global_maa
+}
+
+innovation_dire_troll = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dire
+
+	potential = {
+		this.culture_group = culture_group:troll_group
+	}
+
+	unlock_maa = dire_troll
+
+	flag = global_maa
+}
+
+innovation_dragon = {
```

Comment:
I would add drakonid as unique giants of dragon culture.
https://wowpedia.fandom.com/wiki/Drakonid

Reply:
👌 can I add them to ideas sheet and add them on another cultural maas addition?

Reply:
Mmmkay.